code/modules/cargo/packs.dm

```diff
@@ -617,6 +607,23 @@
 	crate_name = "bulk energy guns crate"
 	crate_type = /obj/structure/closet/crate/secure/plasma
 
+/datum/supply_pack/security/armory/laser
+	name = "Laser gun Single-Pack"
+	desc = "Contains one lethal, high-energy laser gun, Requires Armory access to open."
+	cost = 1000
+	small_item = TRUE
+	contains = list(/obj/item/gun/energy/laser)
+	crate_name = "single laser gun crate"
+
+/datum/supply_pack/security/armory/laser_single
+	name = "Bulk Laser guns Crate"
+	desc = "Contains three lethal, high-energy laser guns. Requires Armory access to open."
+	cost = 2150
```

Comment:
Cost was increased, not documented

Reply:
Ah i forgot to add it in the CL, now added